### ISSUE TITLE
delete/stop: update config fails: yaml: control characters not allowed + gibberish output

### ISSUE THREAD

**Issue by GiuseppeMP** [kubernetes/minikube#5499]
**The exact command to reproduce the issue**: both: minikube delete, stop.

Output: [Video](https://www.dropbox.com/s/7j5jwgkd202dy93/Minikube-ubuntu-19.04%202019-09-30%2010-18.mp4?dl=0)

**The full output of the command that failed**: [full-output.txt
 ](https://www.dropbox.com/s/wpesaisg55gwm3m/full-output-detailed.txt?dl=0)

</details>

**The output of the `minikube logs` command**:<details>

```
➜ minikube logs                                 

💣  command runner: getting ssh client for bootstrapper: Error dialing tcp via ssh client: dial tcp 127.0.0.1:46735: connect: connection refused

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new/choose
```

</details>

**The operating system version**: Distro: Ubuntu 19.04 (Disco Dingo) 

```System:    Host: giuseppemp-meta Kernel: 5.0.0-29-generic x86_64 bits: 64 Desktop: KDE Plasma 5.15.4 
           Distro: Ubuntu 19.04 (Disco Dingo) 
Machine:   Type: Laptop System: Dell product: Latitude 3480 v: N/A serial: <root required> 
           Mobo: Dell model: 06M7PJ v: A00 serial: <root required> UEFI: Dell v: 1.12.0 date: 05/27/2019 
Battery:   ID-1: BAT0 charge: 33.2 Wh condition: 33.2/42.0 Wh (79%) 
CPU:       Topology: Dual Core model: Intel Core i5-6200U bits: 64 type: MT MCP L2 cache: 3072 KiB 
           Speed: 1593 MHz min/max: 400/2800 MHz Core speeds (MHz): 1: 1600 2: 1587 3: 1590 4: 1580 
Graphics:  Device-1: Intel Skylake GT2 [HD Graphics 520] driver: i915 v: kernel 
           Display: x11 server: X.Org 1.20.4 driver: modesetting unloaded: fbdev,vesa resolution: 1366x768~60Hz 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 520 (Skylake GT2) v: 4.5 Mesa 19.0.8 
Audio:     Device-1: Intel Sunrise Point-LP HD Audio driver: snd_hda_intel 
           Sound Server: ALSA v: k5.0.0-29-generic 
Network:   Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet driver: r8169 
           IF: enp2s0 state: down mac: 18:66:da:ff:0c:61 
           Device-2: Qualcomm Atheros QCA6174 802.11ac Wireless Network Adapter driver: ath10k_pci 
           IF: wlp3s0 state: up mac: d4:6a:6a:ef:f6:45 
           Device-3: Qualcomm Atheros type: USB driver: btusb 
           IF-ID-1: br-3fe5fe1ef0b7 state: down mac: 02:42:ca:b8:dc:99 
           IF-ID-2: br-7ec7bf7635e9 state: down mac: 02:42:52:ff:c7:f3 
           IF-ID-3: br-a30332070a2c state: down mac: 02:42:32:df:64:24 
           IF-ID-4: br-cfe2f3176d02 state: up speed: N/A duplex: N/A mac: 02:42:7e:28:0a:50 
           IF-ID-5: br-da69c95f9b50 state: down mac: 02:42:83:34:a1:b1 
           IF-ID-6: docker0 state: down mac: 02:42:07:8f:8a:31 
           IF-ID-7: vboxnet0 state: down mac: 0a:00:27:00:00:00 
           IF-ID-8: vethf800386 state: up speed: 10000 Mbps duplex: full mac: d6:b8:e1:d0:0b:e3 
Drives:    Local Storage: total: 223.58 GiB used: 188.12 GiB (84.1%) 
           ID-1: /dev/sda vendor: Western Digital model: WDS240G2G0A-00JH30 size: 223.58 GiB 
Partition: ID-1: / size: 178.62 GiB used: 155.31 GiB (86.9%) fs: ext4 dev: /dev/sda3 
           ID-2: swap-1 size: 7.54 GiB used: 0 KiB (0.0%) fs: swap dev: /dev/sda2 
Sensors:   System Temperatures: cpu: 59.0 C mobo: 56.0 C sodimm: 37.0 C 
           Fan Speeds (RPM): cpu: 0 
Info:      Processes: 264 Uptime: 1h 30m Memory: 15.25 GiB used: 5.39 GiB (35.3%) Shell: zsh inxi: 3.0.33 
```

**Comment by jadia** [comment]
I'm having the exact same issue with `minikube stop`

It works but also ends up printing some gibberish on the terminal.

**Comment by GiuseppeMP** [comment]
> I'm having the exact same issue with `minikube stop`
> 
> It works but also ends up printing some gibberish on the terminal.

Hey There, btw, beforewards that bug happend, did u updated some bin like kubectl?

**Comment by tstromberg** [comment]
What.. the heck is that?

I saw the note about YAML in the error message, which says to me that this is probably related to updating `~/.kube/config` in some way.

**Comment by medyagh** [comment]
do you mind sharing what version of minikube are you using ? and output

`ls -lah ~/.kube/config`

**Comment by tstromberg** [comment]
I wonder if #5439 may have fixed this.

**Comment by GiuseppeMP** [comment]
> do you mind sharing what version of minikube are you using ? and output
> 
> `ls -lah ~/.kube/config`

```
minikube version: v1.4.0
commit: 7969c25a98a018b94ea87d949350f3271e9d64b6
```

**Comment by GiuseppeMP** [comment]
Using minikube v1.3.1 doesn't  happen.

**Comment by medyagh** [comment]
do you mind sharing the output of this command
```ls -lah ~/.kube/config```
and aslo when you open the ~/.kube/config file does look readable?

**Comment by skonto** [comment]
I see this error consistently on ubuntu 18.04 with minikube 1.4.0

**Comment by tstromberg** [comment]
We're going to have a v1.4.1 release in the next day, I'll be curious to know if it fixes the issue for folks due to #5439

**Comment by GiuseppeMP** [comment]
> @GiuseppeMP
> 
> do you mind sharing the output of this command
> `ls -lah ~/.kube/config`
> and also when you open the ~/.kube/config file does look readable?
> I have a suspicion that your kubeconfig file is either not accessible or corrupted .

Sure, it follows::
<details>
<summary>ls -lah ~/.kube/config</summary>

cat ~/.kube/config

apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM2akNDQWRLZ0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFtTVNRd0lnWURWUVFEREJ0dmNHVnUKYzJocFpuUXRjMmxuYm1WeVFERTFOams1TmpNMk1qTXdIaGNOTVRreE1EQXhNakV3TURJeVdoY05NalF3T1RJNQpNakV3TURJeldqQW1NU1F3SWdZRFZRUUREQnR2Y0dWdWMyaHBablF0YzJsbmJtVnlRREUxTmprNU5qTTJNak13CmdnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURPcTJSZDlRa1BsTWR2R0FlNDhUL0UKTkFuQzRlRCsra1h4NzI3VDlIMGdORHk4bjBSdmVnUEY4eEg2Uis4cFltVWdrNHlxbythRlNOdjZEVlMvSUVabgprRGR4eUZvSGEzdnBOVUM0UllNdVVqU1VBS3VPL0xFS2dDSzBEbWhvRUczbjNmckNuZnNjOWxjejRNL0xJQmxGCkZrc1M4RVlxQ2JEU1BOVEN5NEhKanhCdkZ0OUZXZ3BOWnR0VkRPWVJMUFBlY05Ta0pUVCtnOW1Qb3M5NnJJWk8Kbkk5eFFmTXJaNHYveUd1aVowLzZQdlFzSHpvTTJYRXh6bExoSHhheWJTZWVadjNtWE9tcEl4NHg2OHMzeTZJLwp5cGxvM1lTVWlDUDhGSXVQZnVlSjVvcVVmbXUwQ0daTTNVNG9BakVhL3kwTDJlRmJ6dEFoamVzM2J3MG8vMkFWCkFnTUJBQUdqSXpBaE1BNEdBMVVkRHdFQi93UUVBd0lDcERBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUEwR0NTcUcKU0liM0RRRUJDd1VBQTRJQkFRQzBxN1pGN3pHakt1dVgrb2t1QUw5QzhCU21lSWtBRW9VWXJHRUl3ZWt5K0FLRgpzNEZZKzF3T3J5WTF2RFl6WDR5NStTZnVvcUZYeGZiRjZkOWdhTERNWDB4UktMVkd2dmVZRzJYRHQ5b2g0WDluCkpTUUg3U25DTGJlUG5MU0ZjeDlucHZRRkJvYmRaT0h2VjJvYzViVkwxc1dlZWsvSmdGVTVUcThaV1crMmVGNUIKajNMMnVpNGduek4rZDlQNm5UcDU0cEFtTTh6WSsyWnpaUWNIeTdlNU1qeUpEaEU3cDlzRzlPZGpGZmRxYWtSagphelUzMnFQUzg4emdnUWFZMXNwb1h5TXJHZWlTb2oxMGx5bHNzd21COHVjck1HaUMwWFU5NlpUdVA3RnEyY3NmCmtKL2tyQjlPT3UrRCs2M3V0RGFBc3FDWWxETU90cXdqa0xrY2FEdUUKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
    server: https://192.168.42.119:8443
  name: 192-168-42-119:8443
- cluster:
    certificate-authority: /home/giuseppemp/.minikube/ca.crt
    server: https://192.168.99.105:8443
  name: minikube
contexts:
- context:
    cluster: minikube
    user: minikube
  name: minikube
- context:
    cluster: 192-168-42-119:8443
    namespace: myproject
    user: developer/192-168-42-119:8443
  name: minishift
- context:
    cluster: 192-168-42-119:8443
    namespace: myproject
    user: developer/192-168-42-119:8443
  name: myproject/192-168-42-119:8443/developer
- context:
    cluster: 192-168-42-119:8443
    namespace: myproject
    user: system:admin/192-168-42-119:8443
  name: myproject/192-168-42-119:8443/system:admin
current-context: ""
kind: Config
preferences: {}
users:
- name: developer/192-168-42-119:8443
  user:
    token: kqxQGZ8cRTiRzUZArX_Y2nJR4WB69Yb1wVvAYgfnSGg
- name: minikube
  user:
    client-certificate: /home/giuseppemp/.minikube/client.crt
    client-key: /home/giuseppemp/.minikube/client.key
- name: system:admin/192-168-42-119:8443
  user:
    client-certificate-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURKRENDQWd5Z0F3SUJBZ0lCQ0RBTkJna3Foa2lHOXcwQkFRc0ZBREFtTVNRd0lnWURWUVFEREJ0dmNHVnUKYzJocFpuUXRjMmxuYm1WeVFERTFOams1TmpNMk1qTXdIaGNOTVRreE1EQXhNakV3TURJMFdoY05NakV3T1RNdwpNakV3TURJMVdqQk9NVFV3RlFZRFZRUUtFdzV6ZVhOMFpXMDZiV0Z6ZEdWeWN6QWNCZ05WQkFvVEZYTjVjM1JsCmJUcGpiSFZ6ZEdWeUxXRmtiV2x1Y3pFVk1CTUdBMVVFQXhNTWMzbHpkR1Z0T21Ga2JXbHVNSUlCSWpBTkJna3EKaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUE1aUtqMTJ2YW92MWVtMjdhK2hodjltcGo4MnFiUE9YaQpmQUwxVDBZRElsM3V0YWdpS0pNUTVRaC96Z2FpTUtlRDlHVk5xVERuOVBZWUtYZjB3MWhKOFBsVlljeHlGbDF0CmlPR2hGQ3VrNkV4R2lUZWxhNS82VmZ5NmpTWjVsaEJVL01iNHY5WVhhQ01CaVA1c2F5Q0tyZy9EL3hGQVZqbGwKbXNQYUpYck0zejdPdkZSTlV5b2d3RnQ1cmpqMnN4OG5PRmdJQk41aFdrSjliK1lQcUdLTXRtdVRTSyszdHhTdQo4VWRxVVJtcXBVZ1pjNi9DTVhPbkFnN0U1RzNXRlA0Rkw0Y09FOVV0WHhmbGM1WGFkamJRVlI1dU5oUUk3VkpVClY4TmlXOVhISE4zbVdiMEJuRTMrUkJvM0VpU2NKR0JrQkRxenB3VlZpMHAzbllscmZwSGJZUUlEQVFBQm96VXcKTXpBT0JnTlZIUThCQWY4RUJBTUNCYUF3RXdZRFZSMGxCQXd3Q2dZSUt3WUJCUVVIQXdJd0RBWURWUjBUQVFILwpCQUl3QURBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQVJka3dDL2tYQnJPUjdlSU1oc05JdFM2S3dZQzlJM2lYCkducWxMNjRhWFR3L2ZnL1l3dkJGRmhDV1FFRTNQSFFXSHo3anlFL2ZDSm9KK3NRYXN4Sm1NYkx6eGNVZkV1V3AKMDc0LzlUN0lLblBKSERnQnl0QTI2NGN0QjRxOFlCOG5ncFljMWtzaGlvUWFRSytMU0s1YTA0YUpIQU9EY3ViZwo2aFMrRlRRSXB4V3ZGSU9kUFFJT0MvdndhK29wRXZCbm1LNklJRDJ1TTN0Qjc2dytzWEk2eTlLdlpQcDZVODRWClFuTG9iNW1qaVRudUVmYjZOTUt0V2MvaUozVVk3eCt5K3hSaExyak9SbGNsckNJYnFuVC83amk4RkpzNy9OeWsKNGlCaytCQ1pwVU54ZkM0SXR5SkoxZzVaMEJ0dTB2Q21JQllRL2h6UDlhWVZxU09XZDJnU0ZBPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBNWlLajEydmFvdjFlbTI3YStoaHY5bXBqODJxYlBPWGlmQUwxVDBZRElsM3V0YWdpCktKTVE1UWgvemdhaU1LZUQ5R1ZOcVREbjlQWVlLWGYwdzFoSjhQbFZZY3h5RmwxdGlPR2hGQ3VrNkV4R2lUZWwKYTUvNlZmeTZqU1o1bGhCVS9NYjR2OVlYYUNNQmlQNXNheUNLcmcvRC94RkFWamxsbXNQYUpYck0zejdPdkZSTgpVeW9nd0Z0NXJqajJzeDhuT0ZnSUJONWhXa0o5YitZUHFHS010bXVUU0srM3R4U3U4VWRxVVJtcXBVZ1pjNi9DCk1YT25BZzdFNUczV0ZQNEZMNGNPRTlVdFh4ZmxjNVhhZGpiUVZSNXVOaFFJN1ZKVVY4TmlXOVhISE4zbVdiMEIKbkUzK1JCbzNFaVNjSkdCa0JEcXpwd1ZWaTBwM25ZbHJmcEhiWVFJREFRQUJBb0lCQVFDYktSdkNNRXVXdkxFKwpwcjBwQmJUbk5NL3kzQnVTTVlSRURGS01naTIyekJJWGxDYURxTFFwVjFPaitzRDdnTlhzdnloZlBFUUgycXhyCnVSN2pYZkZzY1E3cUEwZWdZSmxyUW9TeDdzTWVnOTdDd1N2Y3BsM0gvNzlmVDJPR2s3aFpQV1dWSWFsaVlzZTAKTDZXMWpKa3p4aGxsTGFKdEFBSExHT1ZKZzhRQmR6VjdzS3JwSEFBQ1JCME9vNktBdDFlZXRwcmpqRjNObUZGRAp1Q2xBaHZqUkRTbFpuYnRFRHI4aWZMNWNIN0o5bENrUW9EaElQakdIbEYrQkZ3b201SVpES21zb1hHNDc3Ylh4CmVoenM2WnBXUjRNb1YyWC9lY1pJcTlDQy9Ya2ZkSlh1MDRaV1c5NmNnRytYYTlNRmdmVnI4ckFFMGZjenNPREQKbHZjQWJIMEpBb0dCQU9hS1FuYWNKVElpV1pHUnByWVhPVVpWcHp2SlVBb3ZpZllJakgzZ081d29MRCtCRUNYcQpGVThyWlNIUDB1Vk53a0twZktQZmpRWFhrbFZUdVdFV3Bud25hV0NvNDdNL2VtNkgyWnhrL3ZzWnkrMG9kZFlYCmRIc2FXTWdKcktwcnV3QlpEOXlZZ29xMUFsZ2xLVFM5UzRXWVN5aGNveTRXb0IwNTVMdnRHcStqQW9HQkFQK00KNytKY2pTS2hwU3BMVzIwRmVMTkJDNXhDanlIWGk4QVR6dVhNUnFLTW5Sb0RIa1crRDdTVjRNdU5vTllGTThqNQpId0pTZDBheGFQeGp5Z0NaUXVRMjc4UXVzKzdhU2NJTzRTaFN5YnZuQTJ4cXV4NWo1UG5wNnN4NllXY2xwaXpvCncxekpHbnU1MVVFaExNeGhCQUQ1eWEwK2szSmJ5WEI3ME85Qmlla3JBb0dBRUFYdUJxWmxvMjFsd2t0WVJmR3AKUm0wdm1XVXRqZmV0NUE3R0RQMm54dnUzejVZZDFEVEgxRi9Vd0p2Y01abk05YXNvQVkvSkVtM2dPdFA5MzNTZQpqNlNCbTNsQmtqVElRcVJvaUlrd2U2MVlVaHhzekR1SEdnTkR4bXQ5dDc2VEdid1c4U01jZDUyTEV5dEtDT05iCk8yYlJXblhOMWFjMjY3RVpaOEpjQjNjQ2dZQStyNGU0Y3hON3gwTUltT3VZVnB0QS9WZW53UHZ6UlRoV3FZVTcKQVZ6aVNyUE95YTBIVnBQcnJFZzBKazJwc2lUOXkzNDd4NWJTbEhvNFR4SWlDM1pKSE9DN3N1a1lVaHQ5cXNDNApiM1prelVvZHNpQ3NWN3kzUVJZRGlwR1NCNzQzamZFK05XWjFxeVFRa2Q2M1U2VHp5RkljV1JrNkdDSFZvY21pCnB1VDVlUUtCZ0NrVTQ3anVFNko5dno5cWFuVnpZazhNT1FrRGthd1doakNEWXl6dGlhczVhVlY4ejJseDJ3NTgKSnZMVFRvYUErdDBOUzlpNzZUb2tRcGZ5SEh0dXpFaDBWTXdPaU1SM3ZGTlV5NTVlTWJ5QzhhZFNMYjNGcGRZawpCMmV0MGRIQ3BnRUZEOEppaDJzcWp2TzlzZTdUQkx6Szl3cTFCUEpIQU1TbjRwTmZiL3BDCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==
</details>


> 
> and if the 1.3.1 works fine...there must be something we have done in file locking ...
> 
> does this error in 1.4.0 happen consistently or happens sometimes?


Yes, always, every ```minikube stop/delete``` command.

**Comment by tstromberg** [comment]
@GiuseppeMP - I believe this may be fixed by minikube v1.5.0-beta.0 - do you mind testing it? 

https://github.com/kubernetes/minikube/releases/tag/v1.5.0-beta.0

**Comment by GiuseppeMP** [comment]
> @GiuseppeMP - I believe this may be fixed by minikube v1.5.0-beta.0 - do you mind testing it?
> 
> https://github.com/kubernetes/minikube/releases/tag/v1.5.0-beta.0

Hey @tstromberg, it's working properly!! 

![image](https://user-images.githubusercontent.com/4944009/67252454-68d39f00-f449-11e9-87b4-a9542d93bf4b.png)

**Comment by tstromberg** [comment]
Excellent news! Will wait for the v1.5.0 release before closing.
